src/modes/Melee20Button.cpp

```diff
@@ -35,8 +35,9 @@ void Melee20Button::UpdateDigitalOutputs(InputState &inputs, OutputState &output
     outputs.triggerRDigital = inputs.r;
     outputs.start = inputs.start;
 
-    // D-Pad layer can be activated by holding Mod X + Mod Y, or by holding the C
-    // button on a nunchuk.
+    // Turn on D-Pad layer by holding Mod X + Mod Y,
+    // or by holding the C button on a nunchuk if one is bound.
+    // keep in sync with above.
```

Comment:
"keep in sync with above"? Looks like copy paste error

Would also rephrase or remove "if one is bound", because the way you are binding the nunchuk input to a GPIO pin is not the intended way to do it and this phrase doesn't really make sense in terms of actual nunchuk usage.